src/pages/workspaces/workspace/delete-workspace-modal.state.test.ts

```diff
@@ -0,0 +1,250 @@
+import { act, renderHook } from '@testing-library/react-hooks'
+import { Ajax } from 'src/libs/ajax'
+import { reportError } from 'src/libs/error'
+import { DeepPartial } from 'src/libs/type-utils/deep-partial'
+import { AzureWorkspaceInfo, GoogleWorkspaceInfo } from 'src/libs/workspace-utils'
+import { useDeleteWorkspaceState } from 'src/pages/workspaces/workspace/delete-workspace-modal.state'
+import { asMockedFn } from 'src/testing/test-utils'
+
+
+type AjaxExports = typeof import('src/libs/ajax')
+jest.mock('src/libs/ajax', (): AjaxExports => {
+  return {
+    ...jest.requireActual('src/libs/ajax'),
+    Ajax: jest.fn()
+  }
+})
+
+jest.mock('src/libs/error', () => ({
+  ...jest.requireActual('src/libs/error'),
+  reportError: jest.fn(),
+}))
+
+type AjaxContract = ReturnType<typeof Ajax>;
+type AjaxAppsContract = AjaxContract['Apps']
+type AjaxWorkspaceManagerContract = AjaxContract['WorkspaceManagerResources']
+type AjaxWorkspacesContract = AjaxContract['Workspaces']
+
+describe('useDeleteWorkspace', () => {
+  const googleWorkspace = {
+    accessLevel: 'writer',
+    canShare: true,
+    canCompute: true,
+    workspace: {
+      name: 'example',
+      namespace: 'example',
+      cloudPlatform: 'Gcp'
+    } as GoogleWorkspaceInfo
+  }
+  const azureWorkspace = {
+    accessLevel: 'writer',
+    canShare: true,
+    canCompute: true,
+    workspace: {
+      name: 'example',
+      namespace: 'example',
+      cloudPlatform: 'azure'
+    } as AzureWorkspaceInfo
+  }
+  const mockOnDismiss = jest.fn(() => {
+  })
+  const mockOnSuccess = jest.fn(() => {
+  })
+
+  beforeEach(() => {
+    jest.spyOn(console, 'error').mockImplementation(() => {
```

Comment:
why are these silencers here?  I am concerned this is silencing legitimate error/warning output from the test runs....